packages/nodes-base/nodes/Zoho/descriptions/PurchaseOrderDescription.ts

```diff
@@ -86,11 +91,12 @@ export const purchaseOrderFields = [
 				],
 				operation: [
 					'create',
+					'upsert',
 				],
 			},
 		},
 	},
-	makeProductDetails('purchaseOrder', 'create'),
+	makeProductDetails('purchaseOrder', 'create', { hasUpsert: true }),
```

Comment:
This UI displayOptions misses upsert in the show section (the additional fields are not found by the code).